.drone.yml

```diff
@@ -109,9 +109,11 @@ pipeline:
       GS_CLIENT_EMAIL: ${GS_CLIENT_EMAIL}
       GS_PRIVATE_KEY: ${GS_PRIVATE_KEY}
     commands:
-      - make mark
-      - ui/build-ui-plugins.sh
-      - make sincemark
+      - mkdir -p $BIN/ui
+      - wget -nv $(gsutil ls -l "gs://vic-ui-builds" | grep -v TOTAL | grep vic_ | sort -k2 -r | (trap " " PIPE; head -1) | xargs | cut -d " " -f 3 | sed "s/gs:\/\//https:\/\/storage.googleapis.com\//") -P $BIN
```

Comment:
This is getting a bit crazy with the pipes, completely impossible to debug. Can you break this up a bit so we can at least see some of the steps along the way?

Reply:
I know. All of that just returns a single path so I'm not sure if I can realistically break it up. But any suggestions on how to do that I am open to. It was taken directly from here: https://github.com/vmware/vic-product/blob/master/installer/build/build.sh#L109

Reply:
cutting it between the trap and xargs will at least provide the output from the gsutil command pre-parsing.